exercises/practice/binary-search/.docs/instructions.md

```diff
@@ -1,24 +1,28 @@
 # Instructions
 
-Implement a binary search algorithm.
+Your task is to implement a binary search algorithm.
 
-Searching a sorted collection is a common task.
-A dictionary is a sorted list of word definitions.
-Given a word, one can find its definition.
-A telephone book is a sorted list of people's names, addresses, and telephone numbers.
-Knowing someone's name allows one to quickly find their telephone number and address.
+A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
+It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
 
-If the list to be searched contains more than a few items (a dozen, say) a binary search will require far fewer comparisons than a linear search, but it imposes the requirement that the list be sorted.
+```exercism/caution
+Binary search only works when a list has been sorted.
+```
 
-In computer science, a binary search or half-interval search algorithm finds the position of a specified input value (the search "key") within an array sorted by key value.
+The algorithm looks like this:
 
-In each step, the algorithm compares the search key value with the key value of the middle element of the array.
+- Divide the sorted list in half and compare the middle element with the item we're looking for.
```

Comment:
We could emphasize the "sorted" as it is a requirement of the algorithm.

```suggestion
- Divide the _sorted_ list in half and compare the middle element with the item we're looking for.
```

Reply:
We have an `exercism/caution` block right above. I _think_ (but am not 100% confident) that this should be enough to not need emphasizing within the sentence itself.

What do you think?

Reply:
It could very well be enough if it is there.  It is also repeated, and so the repetition itself is an emphasis.  A little bit of markdown for emphasis could help to bring that up as being an important requirement though.  If we were speaking, I would inflect on that single word, just a little bit.

Reply:
This is what it looks like on the site, for context.

<img width="873" alt="Screen Shot 2023-03-08 at 12 12 45 PM" src="https://user-images.githubusercontent.com/276834/223698845-14579bbd-63d7-42d7-9267-a37cc1efdc91.png">